bundles/org.openhab.voice.openaitts/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="voice:openaitts">
+		<parameter-group name="authentication">
+			<label>Authentication</label>
+			<description>Authentication for connecting to OpenAI API.</description>
+		</parameter-group>
+		<parameter-group name="tts">
+			<label>TTS Configuration</label>
+			<description>Configure Text to Speech.</description>
+		</parameter-group>
+		<parameter name="apiKey" type="text" required="true" groupName="authentication">
+			<label>API key</label>
+			<required>true</required>
+			<description>OpenAI API key.</description>
+		</parameter>
+		<parameter name="apiUrl" type="text" required="true" groupName="authentication">
+			<label>Api URL</label>
+			<required>true</required>
+			<description>TTS host API URL.</description>
+			<default>https://api.openai.com/v1/audio/speech</default>
+		</parameter>
+		<parameter name="model" type="text" required="true" groupName="tts">
+			<label>Model</label>
+			<required>true</required>
+			<description>ID of the model to use.</description>
+			<default>tts-1</default>
+		</parameter>
```

Comment:
Can you also add the available options for this parameter so it can be selected in the UI?